src/Tickets/Commerce/Shortcodes/Checkout_Shortcode.php

```diff
@@ -115,10 +108,22 @@ public function get_gateways_active() {
 	 * @return int The number of connected gateways.
 	 */
 	public function get_gateways_connected() {
+		_deprecated_function( __METHOD__, 'TBD', __CLASS__ . '::get_gateways_available' );
+		return $this->get_gateways_available();
```

Comment:
Let's not deprecate this as we may want to keep the method to know how many gateways are connected. Just have the available in a new one and that's it.